crowbar_framework/Gemfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-source "https://rubygems.org"
+source "http://rubygems.org"
```

Comment:
If we're only doing it because SLE11 is broken, then I suggest to wait for the move to SLE12. Is there any other reason?

(it's bad to go back to http)